modules/planning/tasks/deciders/open_space_decider/open_space_roi_decider.cc

```diff
@@ -298,20 +298,21 @@ void OpenSpaceRoiDecider::SetPullOverSpotEndPose(Frame *const frame) {
 }
 
 void OpenSpaceRoiDecider::SetParkAndGoEndPose(Frame *const frame) {
-  const double kSTargetBuffer = 5.0;
-  const double kSpeedRatio = 0.8;  // after adjust speed is 80% of speed limit
+  const double kSTargetBuffer = 8.0;
+  const double kSpeedRatio = 0.3;  // after adjust speed is 80% of speed limit
```

Comment:
1. modify the comments accordingly to be consistent,

Reply:
Fixed. Thank you.